junit-platform-launcher/src/main/java9/org/junit/platform/launcher/listeners/FlightRecordingListener.java

```diff
@@ -112,27 +120,38 @@ public void executionFinished(TestIdentifier test, TestExecutionResult result) {
 	@Name("org.junit.Test")
 	@Label("Test")
 	static abstract class TestEvent extends Event {
-
 		@Label("Unique Id")
 		String uniqueId;
-
 		@Label("Display Name")
 		String displayName;
 	}
 
 	@Name("org.junit.DynamicTestRegistration")
+	@Label("Dynamic Test Registration")
```

Comment:
I don't think stack traces are useful for these events as they would only point back into the test framework. Adding the annotation @StackTrace(false) will turn them off.

Reply:
Good point -- really need to read https://docs.oracle.com/en/java/javase/14/jfapi/create-events.html more carefully. That entire event class might be dropped in the near future -- as it is not relevant for test result recording anyway.